packages/git/src/browser/git-view-contribution.ts

```diff
@@ -341,6 +594,42 @@ export class GitViewContribution extends AbstractViewContribution<GitWidget>
                 return this.quickOpenService.clone(url, folder, branch);
             }
         });
+        const commit = (scmRepository: ScmRepository, message: string) => {
+            const localUri = scmRepository.provider.rootUri;
+            if (localUri) {
+                this.doCommit({ localUri }, message);
+            }
+        };
+        registry.registerCommand(GIT_COMMANDS.COMMIT,
```

Comment:
It seems we have `refresh` and `sign off by` already.
![screen shot 2019-02-28 at 11 00 16](https://user-images.githubusercontent.com/1405703/53558331-1c4cf380-3b48-11e9-8a6b-a03fbcb80670.jpg)

Reply:
Could you please clarify what's the problem?